src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./Navbar.module.css";
+
+import { OutlinedAvatar, BrandLogo } from "assets";
+import { ROUTE_ROOT } from "utils";
+
+export const Navbar = () => {
+  return (
+    <nav className={`align-i-ctr flex justify-c-sb p-1 ${styles.Navbar}`}>
+      <div className="flex">
+        <BrandLogo className={styles.Logo} />
```

Comment:
yeah, actually to avoid class name clashes this time I have decided to use CSS modules. And in CSS modules, we add classes like this.

you can read more here, https://create-react-app.dev/docs/adding-a-css-modules-stylesheet